src/test/java/cz/muni/fi/pa165/library/services/BookServiceTest.java

```diff
@@ -0,0 +1,28 @@
+package cz.muni.fi.pa165.library.services;
+
+import cz.muni.fi.pa165.library.entities.Book;
+import cz.muni.fi.pa165.library.repositories.BookRepository;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.test.context.junit4.SpringRunner;
+
+/**
+ * @author Katarína Hermanová
+ * UČO 433511
+ * Github katHermanova
+ */
+@RunWith(SpringRunner.class)
+public class BookServiceTest {
+    @Autowired
+    private BookService bookService;
+    @MockBean
+    private BookRepository bookRepository;
+    private Book book;
+
+    @Bean
```

Comment:
You define beans in configuration classes. Also such simple beans are created automatically by components scan.